lib/chef/resource/windows_firewall_profile.rb

```diff
@@ -166,12 +166,7 @@ def firewall_enabled?(profile_name)
                 return $true
             } else {return $false}
           CODE
-          firewall_status = powershell_out(cmd).stdout
-          if /True/.match?(firewall_status)
-            true
-          elsif /False/.match?(firewall_status)
-            false
-          end
+          powershell_exec!(cmd).result
```

Comment:
Does the boolean parsing just go away with our better JSON parsing from powershell_exec?

Reply:
yes. They are parsed as true booleans.

Reply:
😄